src/Juvix/Compiler/Backend/Geb/Language.hs

```diff
@@ -144,6 +216,74 @@ instance HasAtomicity Object where
     ObjectHom {} -> Aggregate appFixity
     ObjectInteger -> Atom
 
+instance HasAtomicity Expression where
+  atomicity = \case
+    ExpressionMorphism m -> atomicity m
+    ExpressionObject o -> atomicity o
+
+instance HasAtomicity TypedMorphism where
+  atomicity _ = Aggregate appFixity
+
+instance Show Morphism where
```

Comment:
I think the `Show` instances should be removed, because they duplicate the functionality of pretty printing.

Reply:
This was for debugging purposes. Removed.

Reply:
Done.